recipes/trottersuzuki/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "trottersuzuki" %}
+{% set version = "1.6.2" %}
+{% set sha256 = "9f5134c54a9d6b44ed7b0422abb724e295fdcac00f6b94b9da25959be07456d9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url:  https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - gcc
+    - numpy
```

Comment:
This package links to numpy, so you'll need to version this [as listed in the docs](https://conda-forge.org/docs/meta.html#building-against-numpy).